package.json

```diff
@@ -1,20 +1,22 @@
 {
-    "name": "live-fanatic-backend",
-    "version": "1.0.0",
-    "description": "",
-    "main": "index.js",
-    "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1",
-        "dev": "nodemon index",
-        "start": "node index",
-        "all": "concurrently \"npm run dev\" \"cd app && npm run dev\""
-    },
-    "author": "",
-    "license": "ISC",
-    "dependencies": {
-        "better-sqlite3": "^7.4.6",
-        "cookie-parser": "^1.4.6",
-        "express": "^4.18.1",
-        "express-session": "^1.17.3"
-    }
-}
\ No newline at end of file
+  "name": "live-fanatic-backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon index",
+    "start": "node index",
+    "all": "concurrently \"npm run dev\" \"cd app && npm run dev\""
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@material-ui/core": "^4.12.4",
```

Comment:
ska inte dessa ligga i app/package.json?

Reply:
Installen resulterade i errors när vi körde den i app/package.json, men gick igenom och fungerar när vi körde den i live-only-fanatic/package.json

Reply:
måste fixas isåfall, backended ska inte ha saker som visas i front-end, vi kommer få problem när den packeteras för release, worst case så får vi vänta med icons tills det e fixat

Reply:
måste fixas isåfall, backended ska inte ha saker som visas i front-end, vi kommer få problem när den packeteras för release, worst case så får vi vänta med icons tills det e fixat

Reply:
Det gick att lösa manuelt, lol. Fixat och åtgärdat. Ska comita och pusah upp så kan du reviewa igen. 

Allt är åtgärdat förutom databas-kopplingen som kommer i nästa steg